### PR TITLE
Use winrm 1.3.x in 0.8-stable

### PR DESCRIPTION
**Pull request by jaym**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in knife-windows.gemspec
 gemspec
 
+gem 'em-winrm', git: 'https://github.com/chef/em-winrm'
```
Comment (adamedx) [inline Gemfile]: This needs to go, right?

**Comment by adamedx** [comment]
Other than the git dep in the gemfile, I'm good on this one.

File: Gemfile (hunk 1 above)
Comment (jaym) [inline Gemfile]: yep

**Comment by ksubrama** [comment]
:+1:

**Pull request merged**